testing/matrix-sdk-integration-testing/src/tests/redaction.rs

```diff
@@ -34,7 +34,14 @@ async fn test_redaction() -> Result<()> {
 
     room.send_state_event(content, "").await?;
     // sync up.
-    tamatoa.sync_once(Default::default()).await?;
+    for _ in 0..2 {
+        // we call sync up to three times to give the server time to flush other
```

Comment:
`0..2` has two elements: 0, and 1.

Reply:
And `0..=2` has three elements, 0, 1 and 2.